cluster/k8s-1.11.0/README.md

```diff
@@ -11,6 +11,7 @@ local machine and are the pushed to a registry which is exposed at
 ```bash
 export KUBEVIRT_PROVIDER=k8s-1.11.0
 export KUBEVIRT_NUM_NODES=2 # master + one nodes
+export KUBEVIRT_MEMORY_SIZE=8196
```

Comment:
why would you add it to all example...they're fine with the default 4GB arent' they? If you need to change that for all (but I think you don't) then it's better to change the default.

Reply:
Sure we can keep there default 4GB. But at least, there is some notice, that user can change memory size.